supernode/daemon/mgr/scheduler/manager.go

```diff
@@ -59,18 +59,18 @@ func (sm *Manager) Schedule(ctx context.Context, taskID, clientID, peerID string
 		return nil, err
 	}
 	if len(pieceAvailable) == 0 {
-		return nil, errors.Wrapf(errortypes.ErrPeerWait, "taskID: %s", taskID)
+		return nil, errors.Wrapf(errortypes.ErrPeerWait, "taskID(%s) clientID(%s)", taskID, clientID)
 	}
-	logrus.Debugf("scheduler get available pieces %v for taskID(%s)", pieceAvailable, taskID)
+	logrus.Debugf("scheduler get available pieces %v for taskID(%s) clientID(%s)", pieceAvailable, taskID, clientID)
 
 	// get running pieces
 	pieceRunning, err := sm.progressMgr.GetPieceProgressByCID(ctx, taskID, clientID, "running")
 	if err != nil {
 		return nil, err
 	}
-	logrus.Debugf("scheduler get running pieces %v for taskID(%s)", pieceRunning, taskID)
+	logrus.Debugf("scheduler get running pieces %v for taskID(%s) clientID(%s)", pieceRunning, taskID, clientID)
 	runningCount := len(pieceRunning)
-	if runningCount >= config.PeerDownLimit {
+	if runningCount >= sm.cfg.PeerDownLimit {
```

Comment:
why to change this? 
I am wondering if the parameters have been redundant.

Reply:
The `config.PeerDownLimit` is a const variable which is the default value.